lang/cs/Org.Apache.REEF.Driver/Bridge/DriverBridgeConfiguration.cs

```diff
@@ -48,10 +48,18 @@ public class DriverBridgeConfiguration : ConfigurationModuleBuilder
         public static readonly OptionalImpl<IStartHandler> OnDriverStarted = new OptionalImpl<IStartHandler>();
```

Comment:
`DriverBridgeComfiguration` is obsolete. We should have this in `DriverConfiguration` as well or even instead.